config/jest/setup.js

```diff
@@ -1,5 +1,8 @@
 'use strict';
 import React from 'react';
+import addons, { mockChannel } from '@storybook/addons';
+
+addons.setChannel(mockChannel());
```

Comment:
🎉  This will allow us to reuse props exported from stories, even if they have knobs or actions from storybook. Thanks for adding this.